container/push.bzl

```diff
@@ -90,6 +90,9 @@ def _impl(ctx):
         tag = tag,
     ))
 
+    if ctx.attr.retry_count > 0:
+        pusher_args.append("-retry-count={}".format(ctx.attr.retry_count))
```

Comment:
Maybe there should be some validation here? Might be confusing if a user provides -1 and nothing happens.

I think we can merge given this change.